devs/docs/es-backports.rst

```diff
@@ -396,11 +396,11 @@ should be crossed out as well.
 - [ ] 6b5cf1b0318 For constant_keyword, make sure exists query handles missing values. (#55757)
 - [ ] e8ef44ce78a Allow Bulk Snapshot Deletes to Abort (#56009) (#56111)
 - [ ] e01b999ef03 Add Functionality to Consistently Read RepositoryData For CS Updates (#55773) (#56091)
-- [ ] 3a64ecb6bfe Allow Deleting Multiple Snapshots at Once (#55474) (#56083)
-- [ ] 69f50fe79f9 Improve same-shard allocation explanations (#56010)
-- [ ] 54dbea6c65c Improve RemoteConnectionManager consistency (#55759)
+- [x] 3a64ecb6bfe Allow Deleting Multiple Snapshots at Once (#55474) (#56083)
+- [x] 69f50fe79f9 Improve same-shard allocation explanations (#56010)
+- [s] 54dbea6c65c Improve RemoteConnectionManager consistency (#55759)
 - [x] a508d3303d5 Ensure no circular reference in translog tragic exception (#55959)
-- [ ] 9eb67365002 Fix NullPointer when message shortcircuited (#55945)
+- [x] 9eb67365002 Fix NullPointer when message shortcircuited (#55945)
```

Comment:
apologies, forgot to check it before, it's been backported with: https://github.com/crate/crate/commit/61b6b3ddfee42e10b617dfa1f16ac15460064908